src/fermilib/utils/_jellium.py

```diff
@@ -379,42 +374,36 @@ def position_potential_operator(n_dimensions, grid_length,
     return operator
 
 
-def jellium_model(n_dimensions, grid_length, length_scale,
-                  spinless=False, momentum_space=True):
+def jellium_model(grid, spinless=False, momentum_space=True):
     """Return jellium Hamiltonian as FermionOperator class.
 
     Args:
-        n_dimensions: An int giving the number of dimensions for the model.
-        grid_length: Int, the number of points in one dimension of the grid.
-        length_scale: Float, the real space length of a box dimension.
-        spinless: Bool, whether to use the spinless model or not.
-        momentum_space: Boole, whether to return in momentum space (True)
+        grid (fermilib.utils.Grid):
+        spinless (bool): Whether to use the spinless model or not.
+        momentum_space (bool): Whether to return in momentum space (True)
             or position space (False).
 
     Returns:
         hamiltonian: An instance of the FermionOperator class.
     """
-    if grid_length % 2 == 0 and (grid_length & (grid_length - 1)) != 0:
+    if grid.length % 2 == 0 and grid.length & (grid.length - 1):
         raise OrbitalSpecificationError(
             'Must use an odd number or a power of 2 for momentum modes.')
 
     if momentum_space:
-        hamiltonian = momentum_kinetic_operator(n_dimensions,
-                                                grid_length,
-                                                length_scale,
-                                                spinless)
-        hamiltonian += momentum_potential_operator(n_dimensions,
-                                                   grid_length,
-                                                   length_scale,
+        hamiltonian = momentum_kinetic_operator(grid, spinless)
```

Comment:
To keep this PR small, other functions will be updated in follow-up PRs. I did one or two in this one so that it was clear how I was going to be using Grid.